src/app/home/home.js

```diff
@@ -1,120 +1,50 @@
-var home = `
-  <div class="home-container">
-  <div class="slideshow-container">
-
-    <div class="mySlides fade">
-      <div class="numbertext">1 / 3</div>
-      <img src="../../../assets/images/offers/offer1.jpg" style="width:100%">
-      <div class="text">Caption Text</div>
-    </div>
-
-    <div class="mySlides fade">
-      <div class="numbertext">2 / 3</div>
-      <img src="../../../assets/images/offers/offer2.jpg" style="width:100%">
-      <div class="text">Caption Two</div>
-    </div>
-
-    <div class="mySlides fade">
-      <div class="numbertext">3 / 3</div>
-      <img src="../../../assets/images/offers/offer3.jpg" style="width:100%">
-      <div class="text">Caption Three</div>
-    </div>
+import { banners } from '../../config/banners';
+import { categories } from '../../config/categories';
 
+const listAllBanners = (banner) => {
+  return `
     <div class="mySlides fade">
-      <div class="numbertext">3 / 3</div>
-      <img src="../../../assets/images/offers/offer4.jpg" style="width:100%">
-      <div class="text">Caption Three</div>
-    </div>
-
-    <div class="mySlides fade">
-      <div class="numbertext">3 / 3</div>
-      <img src="../../../assets/images/offers/offer5.jpg" style="width:100%">
-      <div class="text">Caption Three</div>
+      <img src="${banner.bannerImageUrl}" style="width:100%" alt="${banner.bannerImageAlt}">
     </div>
+  `
+}
 
-    <a class="prev" onclick="plusSlides(-1)">&#10094;</a>
-    <a class="next" onclick="plusSlides(1)">&#10095;</a>
+const listAllCategories = (category) => {
+  return `
+    <li class="category-list">
+      <div class="products-category">
+        <figure class="category">
+          <img src="${category.imageUrl}" alt="${category.key}">
+        </figure>
+        <div class="product-detail">
+          <h5>${category.name}</h5>
+          <p>${category.description}</p>
+          <a class="btn-explore-category" href="#">Explore-${category.key}</a>
```

Comment:
href="#" should have a variable, so that you can change value when you use it. It is also as important as name or description